_sass/blocks/_communities_content.scss

```diff
@@ -17,21 +17,8 @@
   }
 } 
 
-.communities_content-icon img{
-  height: 20px;
-  padding-right: 8px;
-
-  &.iron {
-    height: 16px;
-  }
-
-  &.copper {
-    height: 22px;
-  }
-
-  &.gas {
-    height: 25px;
-  }
+.communities_content-icon i{
```

Comment:
Selector `communities_content-icon` should be written in lowercase with hyphens<br>Opening curly brace `{` should be preceded by one space